src/components/text-editor/text-editor.spec.tsx

```diff
@@ -983,28 +1079,36 @@ describe("TextEditor", () => {
         wrapper.update();
       });
       act(() => {
-        wrapper.find(Editor).props().handleBeforeInput(" ");
+        const { handleBeforeInput, editorState } = wrapper.find(Editor).props();
+
+        handleBeforeInput?.(" ", editorState, 0);
       });
 
       expect(onLinkAdded).toHaveBeenCalledWith(url);
     });
 
     it("renders any EditorLinkPreviews passed in via the `previews` prop", () => {
       const previews = [
-        <EditorLinkPreview />,
-        <EditorLinkPreview />,
-        <EditorLinkPreview />,
+        <EditorLinkPreview key="1" />,
+        <EditorLinkPreview key="2" />,
+        <EditorLinkPreview key="3" />,
       ];
       wrapper = render({ onLinkAdded, previews });
       expect(wrapper.find(EditorLinkPreview).length).toEqual(3);
     });
 
+    it("coverage filler for else path", () => {
```

Comment:
I'm not a big fan of "coverage filler" as a test description. If you're not going to write an actual meaningful test, imo it's more honest to just put an `instanbul ignore else` comment in the code.

Having said that, this test does make an actual assertion, although I can't tell how useful it is as I'm not sure what exactly (and which "else" in particular) is being tested. I'm more than happy to keep this test if it serves a useful purpose other than "filling coverage", but in that case it should have a descriptive name. (If you can't think of a good description of the test to put here then that's probably a sign it's not actually testing anything useful, in which case `istanbul ignore` might be the way to go!)

Reply:
You know what, I wasn't actually meant to leave this as `coverage filler... `. I did that to make debugging the tests I was writing easier and forgot to change it back 😄 

I was battling with `99.98%` coverage for ages and it was the else path that wasn't covered, hence the name of the test. I'll get it changed to something more test-like